hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -297,7 +299,8 @@ message OMResponse {
 
   optional ListTrashResponse                  listTrashResponse            = 91;
   optional RecoverTrashResponse               RecoverTrashResponse         = 92;
-  optional PurgeDirectoriesResponse           purgeDirectoriesResponse     = 93;
+  optional PurgePathsResponse                 purgePathsResponse           = 93; // [deprecated = true]
```

Comment:
Here again the same as above. Should it be without comment line ?

I could see everywhere the deprecated line is commented out.

Reply:
Fixed it